internal/xdscache/v3/cluster_test.go

```diff
@@ -306,12 +306,12 @@ func TestClusterVisit(t *testing.T) {
 			},
 			want: clustermap(
 				&envoy_config_cluster_v3.Cluster{
-					Name:                 "beurocra-7fe4b4/tiny-cog-7fe4b4/443/da39a3ee5e",
-					AltStatName:          "beurocratic-company-test-domain-1_tiny-cog-department-test-instance_443",
+					Name:                 "bureaucr-0796f4/tiny-cog-0796f4/443/da39a3ee5e",
```

Comment:
As tsaarni noted, this PR was written by a human (me). And getting this part right was one of the hardest parts